packages/react-strict-dom/src/native/modules/createStrictDOMComponent.js

```diff
@@ -403,6 +403,8 @@ export function createStrictDOMComponent<T, P: StrictProps>(
        * Resolve the style props
        */
       const inheritedStyles: ?Style = React.useContext(InheritableStyleContext);
+      const inheritedCustomProperties = React.useContext(ThemeContext);
```

Comment:
Since we already read this context here, I'm passing it to `useStyleProps` rather than reading it again from within the hook